### PR TITLE
Task 29 User Service Package and Test

### PR DESCRIPTION
**Pull request by elin3t**
User Service Package and Test

### CODE REVIEW

**Comment by pauriarte** [comment]
+1

File: lib/UserService.pm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    my $userRepository = UserHashRepository->new();
+    my $user = $userRepository->find($username);
+    if ($user)
+    {
+        return $userRepository->delete($username);
+
+    }
+    return 0;
+}
+
```
Comment (manuellerena) [inline lib/UserService.pm:44]: Remove empty lines

**Comment by manuellerena** [comment]
+1

File: lib/UserService.pm (hunk 1 above)
Comment (elin3t) [inline lib/UserService.pm]: I dont get it!
Is a good practice to put a empty line between functions and one at the end of the file, to do it more readable :) 
http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline

Comment (elin3t) [inline lib/UserService.pm:44]: removed!

**Pull request merged**